<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameItem</class>
 <widget class="QWidget" name="GameItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>110</height>
    </size>
   </property>
   <widget class="QLabel" name="pictureBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>PictureBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>0</height>
    </size>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>145</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>gameName</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>11</y>
     <width>195</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
